app/lib/account/consent_backend.dart

```diff
@@ -29,12 +29,25 @@ void registerConsentBackend(ConsentBackend backend) =>
 ConsentBackend get consentBackend =>
     ss.lookup(#_consentBackend) as ConsentBackend;
 
+/// List of consent kinds.
+abstract class ConsentKind {
```

Comment:
nit: If we exposed `ConsentBackend.invitePackageUploader`, etc... instead of exposing `ConsentBackend.invite`, then we wouldn't have to export this. And we could ensure that the `args` is type-safe..

I don't think this is important, we can always cleanup this later. It's not like we have a many implicit invariants here.